pyvista/plotting/plotting.py

```diff
@@ -1752,8 +1752,10 @@ def increment_point_size_and_line_width(self, increment):
 
     def reset_key_events(self):
         """Reset all of the key press events to their defaults."""
-        if hasattr(self, 'iren'):
-            self.iren.clear_key_event_callbacks()
+        if not hasattr(self, 'iren'):
+            return
+
+        self.iren.clear_key_event_callbacks()
```

Comment:
Since all the other `self.add_key_event()` calls are no-ops in case `self.iren` doesn't exist, it seemed more readable to skip the whole method in case `self.iren` is missing. It's not likely that `self` will pick up the attribute mid-execution of the method.